qiskit/visualization/matplotlib.py

```diff
@@ -825,8 +825,14 @@ def _draw_ops(self, verbose=False):
                 else:
                     param_width = 0.0
 
-                if op.name == 'cu1' or op.name == 'rzz' or base_name == 'rzz':
-                    tname = 'U1' if op.name == 'cu1' else 'zz'
+                if (op.name == 'cu1' or op.name == 'cp'
+                        or op.name == 'rzz' or base_name == 'rzz'):
```

Comment:
```suggestion
                if op.name in ['cu1', 'cp', 'rzz'] or base_name == 'rzz':
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5804/commits/54073ddf01439a371ed126e03296a4c61e9bffa3